modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,21 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+TimeUnits are represented by a number followed by the unit: us *(microseconds), ms (milliseconds, default), s (seconds), m (minutes), h *(hours), d (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+[cols="3*"]
+|===
+| ROUTER_THREADS | 4s | Specifies the number of threads for the haproxy router.
```

Comment:
`ROUTER_THREADS` should not be listed here, it is unrelated to timeoute variables, it `Specifies the number of threads for the haproxy router`

Reply:
ack